crates/api_models/src/payments.rs

```diff
@@ -4916,6 +4916,8 @@ pub struct PaymentsDynamicTaxCalculationRequest {
     /// Payment method type
     #[schema(value_type = PaymentMethodType)]
     pub payment_method_type: api_enums::PaymentMethodType,
+    /// Session Id
+    pub session_id: Option<String>,
```

Comment:
session id is mandatory when calling this api right?

Reply:
In case of apple_pay and google we don't need this.
 This is only required for payment_method_type in which we have to call connector for updating the session